src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -113,6 +111,13 @@ public class CaseData implements MappableObject {
     private final LocalDate issueDate;
     private final String generalAppSuperClaimType;
     private final GAMakeApplicationAvailableCheck makeAppVisibleToRespondents;
+    private final String respondentSolicitor1EmailAddress;
+    private final String respondentSolicitor2EmailAddress;
+    private final String respondent1OrganisationIDCopy;
```

Comment:
Where are respondent1OrganisationIDCopy and respondent2OrganisationIDCopy being used?

Reply:
Removed, Thanks for spotting it out.